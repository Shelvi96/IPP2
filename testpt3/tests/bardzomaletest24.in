NEW ID

07;278:8;>44;71;;2;
87846>5::9;:8254
3>1499782
7>94494
3>165258;7
794;3000>93
8;7>4
84036>1039;2:
;2:63>72;200504
45>4002105
478:;305:>446:
64>35107:0;
60:8495>80;6:
@209;;586::668:9841:598:8427:0
@165:6432:50830
@1932;807:84076407;59520;4:
@1100217
@1490694;4;746694338386
@90
@9214;600
@;932569948;8;296271;71022
@5016:07121742005
@381:21328462:
@8:56
@9124628960263205519;11544106
@:37;:360;76394;4449
@58;7533679455483767;15231176
@8338221950434108:74427:;91030
@18284
