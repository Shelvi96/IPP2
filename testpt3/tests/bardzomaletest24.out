45956009312119346
9
115919667461376
0
430526259
0
0
3423954581905
512
1
0
1100100000000000
294976
1100100100000000
50130144990582877
0
